### PR TITLE
#222 VFB-187: Select parcels for day overview and log status change

### PR DESCRIPTION
**Pull request by AmeliaIsCoding**
## What's changed

- Day overview takes in a selection of parcels rather than a date and a collection centre
- Day overview requires at least 1 parcel selected
- The day overview modal now displays an overview of the selected parcels 
- The day overview PDF no longer has subtitles for the date and collection centre
- Carrying out the day overview action changes the status of the parcel to "Day Overview Downloaded"
- Carrying out the day overview action sends audit log(s) to record the status change of the parcel(s)

## Screenshots / Videos
| Before     | After      |
|------------|------------|
|imageimage|imageimage|

## Checklist
- [x] The ticket is up-to-date - Please document any deviations from the original approach if there is any.
- [x] I have documented the testing steps for QA
- [x] I have self-reviewed this PR
- [x] Make sure you've verified it works via `npm run dev`
- [x] Make sure you've verified it works via `npm run build` and `npm run start`
- [x] Make sure you've fixed all linting problems with `npm run lint_fix`
- [x] Make sure you've tested via `npm run test:e2e`

No DB changes.

### CODE REVIEW

**Review by AmeliaIsCoding — COMMENTED** [review]
Self-review

File: src/pdf/DayOverview/DayOverviewPdf.tsx
Comment (rickaw) [inline src/pdf/DayOverview/DayOverviewPdf.tsx:1]: Can we please add a "Collection" column to indicate the method please? ("XXX Church", "Delivery" etc.)

See [here](https://softwire.sharepoint.com/Customers/vauxhallfoodbank/Documents/Generated%20PDFs/day-overview.pdf) for the pdf that's produced currently

File: src/pdf/DayOverview/DayOverviewPdfButton.tsx [hunk 1]
```diff
@@ -85,88 +58,34 @@ const getParcelsOfSpecificDateAndLocation = async (
...
+            return { data: null, error: error };
         }
-
-        const dateString = getCurrentDate(date);
-
-        const acronym = `_${collectionCentreNameAndAbbreviation?.acronym}` ?? "";
-        const location = collectionCentreNameAndAbbreviation?.name ?? "";
-
-        const fileName = `DayOverview_${dateString}${acronym}.pdf`;
+        const fileName = "DayOverview.pdf";
```
[the remark this answers is not in the document]
Comment (rickaw) [reply src/pdf/DayOverview/DayOverviewPdfButton.tsx:84]: I think this is fine!

**Review by rickaw — CHANGES_REQUESTED** [review]

File: src/pdf/DayOverview/DayOverviewPdf.tsx
Comment (AmeliaIsCoding) [reply src/pdf/DayOverview/DayOverviewPdf.tsx:1]: Done

**Pull request merged**